create_remote_nodes.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+source common.sh
+source ocp_install_env.sh
+
```

Comment:
You might want to `set -ex` and include `logging.sh` so we exit on errors and include more verbose output for debugging?

Reply:
Thanks, Steve. I've added these in the new commit.